src/utils.rs

```diff
@@ -0,0 +1,20 @@
+use egui::*;
+
+#[inline(always)]
+pub fn expand_to_pixel(mut rect: Rect, ppi: f32) -> egui::Rect {
```

Comment:
Add documentation for this and the remaining functions in this module

Reply:
> Add documentation for this and the remaining functions in this module
It's for internal use, it's not public API and it was made by the original author, I don't fully understand what it does.

Reply:
Then it might be good to mark these functions as `pub(crate)` instead of `pub`

Reply:
> Then it might be good to mark these functions as `pub(crate)` instead of `pub`

`utils` imported via `mod utils`. So, it's encapsulated inside `lib.rs`.

Reply:
Good point